tests/Riverline/WorkerBundle/Provider/AwsSQSTest.php

```diff
@@ -1,115 +0,0 @@
-<?php
```

Comment:
Can you add in the AwsSQS class something to inform the user that the class is deprecated and no more tested ?

Reply:
Done